man/systemd-coredump.xml

```diff
@@ -88,6 +88,42 @@
       metadata fields in the same way it does for core dumps received from the kernel. In this mode, no core
       dump is stored in the journal.</para>
     </refsect2>
+
+    <refsect2>
+      <title>Core dumps in Containers/Namespaces</title>
+
+      <para>The <filename>systemd-coredump@.service</filename> service will automatically attempt to extract
+      a stacktrace from any core dump as it is generated. For this stacktrace symbols will be resolved based
+      on debug information embedded in the crashing ELF image, or equivalent debug information separately
+      available on the host OS. For processes that crash inside of local containers or other mount
+      namespace-based sandboxes, this debug information is typically not available on the host (simply
+      because containers typically run different software versions than the
+      host). <filename>systemd-coredump</filename> provides two mechanisms to address this:</para>
+
+      <orderedlist>
+        <listitem><para>For full-OS containers (i.e. those that run a proper <filename>init</filename> system
+        as PID 1 inside), it is a good idea to enable <varname>CoredumpReceive=</varname> on the unit (see
+        <citerefentry><refentrytitle>systemd.resource-control</refentrytitle><manvolnum>5</manvolnum></citerefentry>),
+        which ensures that coredumps of a container are attempted to be forwarded to
+        <filename>systemd-coredump@.service</filename> running inside the container, i.e the container gets
+        to process its own core dumps. Note that
```

Comment:
"process and store". The "store" part is important, because all this is not useful if the container is ephemeral.